packages/big-design/src/components/Worksheet/Row/Row.tsx

```diff
@@ -107,6 +107,7 @@ const InternalRow = <T extends WorksheetItem>({ columns, rowIndex }: RowProps<T>
       <RowStatus rowIndex={rowIndex} />
       {columns.map((column, columnIndex) => (
         <Cell
+          action={column.type === 'number' ? column?.action : undefined}
```

Comment:
Should we pass `action` for the `text` type too?

Reply:
Thanks, done